File: src/main/java/org/voegtle/weatherwidget/preferences/WeatherPreferences.java
package org.voegtle.weatherwidget.preferences;

import android.app.ActionBar;
import android.os.Bundle;
import org.voegtle.weatherwidget.base.ThemedActivity;

public class WeatherPreferences extends ThemedActivity {

  @Override
  public void onCreate(Bundle savedInstanceState) {
    super.onCreate(savedInstanceState);

    getFragmentManager().beginTransaction()
        .replace(android.R.id.content, new WeatherPreferenceFragment())
        .commit();

    ActionBar actionBar = getActionBar();
    assert actionBar != null;
    actionBar.setDisplayHomeAsUpEnabled(true);
  }

}

File: src/main/java/org/voegtle/weatherwidget/preferences/WeatherPreferences.kt
package org.voegtle.weatherwidget.preferences

import android.app.ActionBar
import android.os.Bundle
import org.voegtle.weatherwidget.base.ThemedActivity

class WeatherPreferences : ThemedActivity() {

  public override fun onCreate(savedInstanceState: Bundle?) {
    super.onCreate(savedInstanceState)

    fragmentManager.beginTransaction()
        .replace(android.R.id.content, WeatherPreferenceFragment())
        .commit()

    actionBar?.setDisplayHomeAsUpEnabled(true)
  }

}
